content/src/scripts/utils/styling.js

```diff
@@ -1,7 +1,7 @@
-export const applyOpenedPageStyling = () => {
-  document.querySelector("header").style.left = paneWidth();
-  document.querySelector(".nav-sidebar").style.left = paneWidth();
-  document.querySelector("body").style.marginLeft = paneWidth();
+export const applyOpenedPageStyling = (width) => {
```

Comment:
Remove all occurences of paneWidth since not being used

Reply:
Done